src/Common/src/CoreLib/System/Text/DecoderReplacementFallback.cs

```diff
@@ -3,13 +3,16 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Diagnostics;
+using System.Runtime.Serialization;
 
 namespace System.Text
 {
 #if MONO
     [System.Serializable]
-#endif
+    public sealed class DecoderReplacementFallback : DecoderFallback, ISerializable
+#else
     public sealed class DecoderReplacementFallback : DecoderFallback
```

Comment:
@EgorBo the same issue applies to `DecoderReplacementFallbackBuffer` and `EncoderReplacementFallbackBuffer`, this fix isn't complete.

Reply:
@akoeplinger both `DecoderReplacementFallbackBuffer` and `EncoderReplacementFallbackBuffer` are not serializable.

Reply:
@EgorBo right, sorry about that. I was confused by all those similar sounding type names :D